### PR TITLE
Fix asset symbols

### PR DESCRIPTION
**Pull request by Andrei-Masilevich**

### CODE REVIEW

File: libraries/protocol/include/scorum/protocol/config.hpp [hunk 1]
```diff
@@ -13,9 +13,12 @@
 
 #define SCORUM_ADDRESS_PREFIX                  "SCR"
 
+//< SCORUM = SCR with 3 digits of precision
+#define SCORUM_SYMBOL  (uint64_t(3) | (uint64_t('S') << 8) | (uint64_t('C') << 16) | (uint64_t('R') << 24))
+//< VESTS = SP with 6 digits of precision
+#define VESTS_SYMBOL   (uint64_t(6) | (uint64_t('S') << 8) | (uint64_t('P') << 16))
```
Comment (abolinbsu) [inline libraries/protocol/include/scorum/protocol/config.hpp]: May be rename to POWER_SYMBOL?

File: libraries/protocol/include/scorum/protocol/asset.hpp [hunk 2]
```diff
@@ -86,6 +93,11 @@ struct asset
         FC_ASSERT(a.symbol == b.symbol);
         return asset(a.amount + b.amount, a.symbol);
     }
+    friend asset operator*(const asset& a, const asset& b)
```
Comment (alexplevako) [inline libraries/protocol/include/scorum/protocol/asset.hpp]: add unit test for a new function.

Comment (abolinbsu) [reply libraries/protocol/include/scorum/protocol/asset.hpp]: This operation is not correct. SCR*SCR=SCR^2, but operation returns SCR.

File: libraries/protocol/include/scorum/protocol/config.hpp (hunk 1 above)
Comment (Andrei-Masilevich) [reply libraries/protocol/include/scorum/protocol/config.hpp]: To many variables with names vesting, vests. Let it be.

**Pull request merged**